src/extension.test.ts

```diff
@@ -126,6 +126,18 @@ suite('App online file edit validations', () => {
 			problematicContent: '{"invalidProperty":"someValue"}',
 			expectedProblemMessage: 'Incorrect type. Expected "array".',
 		},
+		{
+			filename: 'makecomapp.json',
```

Comment:
wrong file name? makecomapp.schema.json

Reply:
It is correct. It defines the simulated filename, under which the testing file is opened in editor.